.github/workflows/conformance.yml

```diff
@@ -0,0 +1,175 @@
+name: Conformance Test
+
+on:
+  workflow_dispatch:
+    types: [run]
+
+concurrency: # Cancel stale PR builds (but not push builds)
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - os: linux
+            cpu: amd64
+          - os: linux
+            cpu: i386
+          - os: macos
+            cpu: amd64
+          - os: windows
+            cpu: amd64
+          #- os: windows
+            #cpu: i386
+        branch: [version-1-2, version-1-4, version-1-6, devel]
+        include:
+          - target:
+              os: linux
+            builder: ubuntu-18.04
+            shell: bash
+          - target:
+              os: macos
+            builder: macos-10.15
```

Comment:
macos-10.15 also deprecated, please use macos 11 or 12

Reply:
maybe also need to add
```
      - name: 'Install dependencies (macOS)'
        if: runner.os == 'macOS' && matrix.branch == 'devel'
        run: |
          brew install openssl@1.1
          ln -s $(brew --prefix)/opt/openssl/lib/libcrypto.1.1.dylib /usr/local/lib
          ln -s $(brew --prefix)/opt/openssl/lib/libssl.1.1.dylib /usr/local/lib/
```

Reply:
I think the libssl is no longer needed, they solved it upstream afair